src/js/HomeLink.jsx

```diff
@@ -0,0 +1,10 @@
+var React = require('react');
+var Link = require('react-router').Link;
+
+var HomeLink = React.createClass({
+  render: function(){
+    return <Link className="HomeLink" to="/">Home</Link>
```

Comment:
We can remove the `className="HomeLink"` here. We _may_ add another class at some point to add an icon.